types/tape/tape-tests.ts

```diff
@@ -28,6 +28,13 @@ rs = tape.createStream(sopts);
 var htest: typeof tape;
 htest = tape.createHarness();
 
+class CustomException extends Error {
+  constructor(message?: string) {
+    super(message);
+    Object.setPrototypeOf(this, new.target.prototype);
```

Comment:
Maybe we could remove this line just as testing code. then we do not need 
```ts
// TypeScript Version: 2.2
```

Reply:
Good catch. Done.